proposals/acp-specification/acp.ttl

```diff
@@ -0,0 +1,243 @@
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix cc: <http://creativecommons.org/ns#>
+prefix dcterms: <http://purl.org/dc/terms/>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix schema:   <http://schema.org/>
+prefix vann: <http://purl.org/vocab/vann/>
+prefix vcard: <http://www.w3.org/2006/vcard/ns#>
+
+prefix acp: <http://www.w3.org/ns/solid/acp#>
+
+
+acp: a owl:Ontology ;
+    owl:versionInfo "0.1" ;
+    dcterms:title "Solid Access Control Policy Vocabulary"@en ;
+    dcterms:description "A vocabulary to structure policy-based access controls for the Solid ecosystem."@en ;
+    dcterms:creator <https://solidproject.org> ;
+    dcterms:issued "2020-09-15"^^xsd:date ;
+    vann:preferredNamespacePrefix "acp" ;
+    vann:preferredNamespaceUri "http://www.w3.org/ns/solid/acp#" ;
+    cc:attributionURL acp: ;
+    cc:license <https://creativecommons.org/publicdomain/zero/1.0/> .
+
+########################
+#   Named Individuals
+########################
+
+acp:AuthenticatedAgent a rdfs:Resource ;
+    rdfs:label "Authenticated Agent"@en ;
+    rdfs:comment """An IRI representing an authenticated agent.
+ Please note that in a decentralized authentication ecosystem, an authenticated agent may be ephemeral
+ and, from a security standpoint, may not be significantly different than an anonymous agent"""@en ;
+    rdfs:isDefinedBy acp: .
+
+acp:CreatorAgent a rdfs:Resource ;
+    rdfs:label "Creator Agent"@en ;
+    rdfs:comment "An IRI representing the creator of a resource."@en ;
+    rdfs:isDefinedBy acp: .
+
+acp:PublicAgent a rdfs:Resource ;
+    rdfs:label "Public Agent"@en ;
+    rdfs:comment "An IRI representing a public or anonymous agent"@en ;
+    rdfs:isDefinedBy acp: .
+
+
+#################
+#   Classes
+#################
+
+acp:AccessControl a rdfs:Class ;
+    rdfs:label "Access Control"@en ;
+    rdfs:comment "Access Control statements associate an AccessControlResource with specific Policy definitions."@en ;
+    rdfs:isDefinedBy acp: .
+
+acp:AccessControlResource a rdfs:Class ;
+    rdfs:label "Access Control Resource"@en ;
+    rdfs:comment "An RDF resource that includes AccessControl statements"@en ;
+    rdfs:isDefinedBy acp: .
+
+acp:Policy a rdfs:Class ;
+    rdfs:label "Access Policy"@en ;
+    rdfs:comment "A Policy defines a collection of access modes along with the conditions under which the policy is applied."@en ;
+    rdfs:isDefinedBy acp: .
+
+acp:Rule a rdfs:Class ;
+    rdfs:label "Rule"@en ;
+    rdfs:comment "A Rule defines which agent(s), group(s) and client application(s) match a Policy filter"@en ;
+    rdfs:isDefinedBy acp: .
+
+acp:AccessMode a rdfs:Class ;
+    rdfs:label "Access Mode"@en ;
+    rdfs:comment "An abstract access mode. This class should not be used directly. Instead, please use Read, Write and Append"@en ;
+    rdfs:isDefinedBy acp: .
+
+acp:Read a rdfs:Class ;
```

Comment:
Aren't we planning to use `acl:Read`, etc instead of defining an ACP version of these access levels?

Reply:
Absolutely, I need to update the ontology.

Reply:
Updated